src/index.js

```diff
@@ -99,7 +98,7 @@ try {
 
 if (!stat) {
     // file not found, so let's make it
-    mkdirp(outputPath);
+    fs.mkdirSync(outputPath);
```

Comment:
To replace it completely, you need `fs.mkdirSync(outputPath, { recursive: true })`

Reply:
I updated it